executor/sort_test.go

```diff
@@ -58,3 +59,26 @@ func (s *testSuite) TestSortInDisk(c *C) {
 	c.Assert(tk.Se.GetSessionVars().StmtCtx.DiskTracker.BytesConsumed(), Equals, int64(0))
 	c.Assert(tk.Se.GetSessionVars().StmtCtx.DiskTracker.MaxConsumed(), Greater, int64(0))
 }
+
+func (s *testSuite) TestIssue16696(c *C) {
```

Comment:
This test is so much slow.  currently cost nearly 500s.
```
[2020-05-28T03:26:24.163Z] PASS: sort_test.go:63: testSuite.TestIssue16696	496.950s
```
[detail link](https://internal.pingcap.net/idc-jenkins/blue/organizations/jenkins/tidb_ghpr_unit_test/detail/tidb_ghpr_unit_test/37651/pipeline/69)